mineral-ui/mineral-ui
# [Website]: Strip proptypes and flow types from production website

### Description

Strip proptypes and flow types from production website to minimize bundle size

### Motivation and context

Performance. Smaller bundles are better bundles.

_**Website index.js file size decreased from 1.56MB to 1.38MB**_

### Screenshots, videos, or demo, if appropriate

https://remove-proptypes--mineral-ui.netlify.com/

### How to test

1. Compare the size of the generated website `index.js` from [master](https://mineral-ui.com/index.js) vs. from [this branch](https://remove-proptypes--mineral-ui.netlify.com/index.js).
2. Open each of the aforementioned script files and search for propTypes.  Note that in the latter one, propTypes still exist, but are set to empty objects.
2. View the demo and ensure all API doc sections are intact.  These are parsed from flow generated proptypes.

### Types of changes

- Other (provide details below)

Performance optimization

### Checklist
